data/mil2525d/core_data/gdbs/README.md

```diff
@@ -9,62 +9,64 @@ This folder contains the Military Features Core Geodatabase Schema format and an
 
 The following information is intended to succinctly capture the format of the Military Features Schema for other applications that may depend upon this format. 
 
-## Schema Sections
-
-* [Symbol Display Attributes](#symbol-display-attributes)
-* [Modifier (Label) Attributes](#modifier-label-attributes)
-
-## Symbol Display Attributes
+## Graphic Display Amplifiers
 
 The following fields control the display of symbols.
 
-| Attribute Name | Data Type | Range of Values | Default Value/Meaning | Sample Name | Sample Value/Meaning | Explanatory Notes |
-| -------------- | --------- | --------------- | ----------- | -------------------- | ----------------- | ----------------- |
-| identity | long int | 0-9 | 1 (Unknown) | "Friend" | 3 ("friend") | **REQUIRED** |
-| symbolset | short int | 00-99  | (per feature class) | "Air" | 01 ("air") | **REQUIRED** | 
-| entity  | long int | N/A | (per feature class) | "Military (Air) : Fixed-Wing" | "110100"  | **REQUIRED** |
-| context  | short int | 0-2 | 0 (Reality) | "Reality" | 0 ("reality") | Optional |
-| modifier1 | long int | 00-99 | 00 (None) | "Mobility : Air Mobile/Air Assault" | Modifier "01" for Modifier 1 | Optional |
-| modifier2 | long int | 00-99 | 00 (None)  | "Close Range and Support : Casualty Staging" | Modifier "05" for Modifier 2 | Optional |
-| indicator | long int | 0-7 | 0 (None) | Headquarters | Headquarters=2 | Optional (="HQ/TF/FD") |
-| echelon | long int | 0-26 | 0 (None) | Squad  | Squad=12 | Optional |
-| mobility | long int | 0,31-52 (None) | 0 | Rail  | Rail=36 |  Optional |
-| array | long int | 0,61-62 | 0 (None) | Short Towed Array | Short Towed Array=61 | Optional |
-| operationalcondition | long int | 0-5 | 0 (Present) | Planned | Planned=1 | Optional |
-| sidc | string/TEXT | string length(8 or 20) | N/A | "01100110" | SymbolSet:"Air"/"01" + Entity:"100110" | Optional-allows len=20 or len=8(identity=unknown) SIDC format |
+| Attribute Name | 2525 D Field ID | Data Type | Range of Values | Default Value/Meaning | Sample Name | Sample Value/Meaning | Explanatory Notes |
+| -------------- | --------- | --------- | --------------- | ----------- | -------------------- | ----------------- | ----------------- |
+| identity | A | long int | 0-9 | 1 (Unknown) | "Friend" | 3 ("friend") | **REQUIRED** |
```

Comment:
"identity"(affiliation) and "context"(exercise) get mapped to Field "E"  (so does "Battle Dimension" but this is not really specified by an attribute and that must be derived from SymbolSet) - the terms used in Table VII, Field E actually look like the old terms from 2525C.